views/email/breach_notification.hbs

```diff
@@ -0,0 +1,2 @@
+{{!< email }}
+Uh-oh, {{ email }} was involved in {{ breach.name }} data breach.
```

Comment:
Should this be wrapped in a `<p>...</p>` tag, like the strings in email_verify.hbs below?

Reply:
We need to do an entire round of UI design & implementation on these emails. These `.hbs` templates are just the start.